### ISSUE TITLE
Question on PR

### ISSUE THREAD

**Issue by danielwhatmuff** [ngrigoriev/janusgraph-es-aws-authenticator#1]
`For now this module is based on an unreleased (and also unmerged) set of changes extending JanusGraph ES REST API support to support the custom authenticators. This README will be updated once this support is available.`

You mention in the README.md there is a PR waiting to be merged in the main Janusgraph repo. Im guessing that has now been merged?

**Comment by danielwhatmuff** [comment]
Was it this one..? https://github.com/JanusGraph/janusgraph/pull/671

**Comment by ngrigoriev** [comment]
@danielwhatmuff yes, that was the one. Thanks for the ping, I was distracted with other things and did not follow the release process of JG. So, now that code is released and this authenticator should work with 0.3.0+. I'll test it and post an update.

**Comment by ngrigoriev** [comment]
...and should also work with 0.2.2, that code it there too

**Comment by danielwhatmuff** [comment]
Ok thanks for the update.

**Comment by danielwhatmuff** [comment]
Would it be possible to publish a release of this so it can be pulled in easily during builds?

**Comment by ngrigoriev** [comment]
I have added a release, for now done manually. I'll add some simple CI process a bit later.
